doc/MAGIC.md

```diff
@@ -232,7 +233,7 @@ Min and max values must always have the same sign, but it can be negative eg. in
 
 ### Learning Spells
 
-Currently there is only one way of learning spells that is implemented: learning a spell from an item, through a use_action.  An example is shown below:
+Currently there are two ways of learning spells that is implemented: learning a spell from an item(through a use_action) and from spells that have the learn_spells property.  An example of an use item is shown below:
```

Comment:
Traits can also grant spells.  I'm pretty certain it's considered a bug that the spell remains after the trait mutates/purifies away.

Reply:
I didn't know that about traits. This section however is only about learning spells and getting a trait from a spell isn't learning it.

Reply:
```{
    "type": "mutation",
    "id": "SLIME_SPRAY",
    "name": { "str": "Slime Spray" },
    "points": 1,
    "visibility": 1,
    "ugliness": 2,
    "description": "You exude slime from your pores.  You can spread that slime on an opponent in melee range.",
    "spells_learned": [ [ "spell_slime_spray", 1 ] ],
    ```
   I only included it because the language is the same.

Reply:
Thanks for your help. I couldn't find any documentation for spells_learned so I added it to json_info and put a link to that documentation in magic.md since I didn't want to get into mutations in the magic.md document.